src/main/kotlin/lotto/domain/LottoResult.kt

```diff
@@ -0,0 +1,54 @@
+package lotto.domain
+
+class LottoResult(
+    private val _result: MutableMap<Rank, Int> = mutableMapOf()
+) {
+    val result: Map<Rank, Int>
+        get() = _result.toMap()
+
+    init {
+        _result[Rank.THREE] = 0
+        _result[Rank.FOUR] = 0
+        _result[Rank.FIVE] = 0
+        _result[Rank.SIX] = 0
+    }
+
+    fun add(matchCount: Int) {
+        if (Rank.isNotWinningRank(matchCount)) {
+            return
+        }
+        val rank = Rank.of(matchCount)
+        val originalCount = _result[rank] ?: return
+        _result[rank] = originalCount.plus(1)
```

Comment:
```suggestion
        _result[rank] = originalCount + 1
```
`public operator fun plus(other: Int): Int` 이기 때문에 그냥 플러스기호를 사용해주셔도 됩니다.